
Running tests... 
----------------------------------------------------------------------
 test_find_settings (Test_demo.TestAppium.test_find_settings) ... ERROR (18.090003)s

======================================================================
ERROR [18.090003s]: Test_demo.TestAppium.test_find_settings
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mac/jenkins_projects/workspace/mobile_auto_demo/testcases/Test_demo.py", line 33, in test_find_settings
    el1 = driver.find_element(by=AppiumBy.XPATH, value="(//android.widget.ImageView[@resource-id=\"com.sec.android.app.launcher:id/iconview_imageView\"])[2]")
          ^^^^^^
NameError: name 'driver' is not defined

----------------------------------------------------------------------
Ran 1 test in 0:00:18

FAILED
 (Errors=1)

Generating HTML reports... 
reports/TestResults_Test_demo.TestAppium_2024-01-08_13-14-55.html
